logic/tasks/StandardTask.cpp

```diff
@@ -0,0 +1,130 @@
+// Licensed under the Apache-2.0 license. See README.md for details.
```

Comment:
There are two interesting parts in this one, neither related to factories really:
1. runTask and runTaskNonBlocking (should probably be called runTaskAsync or something). They will stay here whatever happens.
2. The others. Some are just wrappers around another task, in which case they can be removed. Others do a bit more (networkGetCached etc.), some more thought will need to be put into them.